frontend/src/text_resources.js

```diff
@@ -208,11 +208,26 @@ let LOCALIZE = new LocalizedStrings({
     emibTest: {
       //Home Page
       homePage: {
+        testTitle: "The Sample eMIB Test",
         welcomeMsg: "Welcome to the eMIB Sample Test"
       },
 
       //HowTo Page
       howToPage: {
+        introductionPage: {
+          title: "Overview",
```

Comment:
Could you double check: I don't think these are used anywhere anymore...

Reply:
Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/435/commits/1f893d658222afa311f49821494f9f6bc659ffae